tests/e2e/performance/io_workload/test_perf_pgsql.py

```diff
@@ -0,0 +1,59 @@
+"""
+Test to run PGSQL performance marker workload
+"""
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.perf_pgsql import PerfPGSQL
+from ocs_ci.framework.testlib import performance, E2ETest
+from ocs_ci.ocs.node import get_node_resource_utilization_from_adm_top
+
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="function")
+def pgsql(request):
+
+    pgsql = PerfPGSQL()
+
+    def teardown():
+        pgsql.cleanup()
+
+    request.addfinalizer(teardown)
+    return pgsql
+
+
+@performance
+@pytest.mark.polarion_id("OCS-2725")
+class TestPGSQLPodPerf(E2ETest):
```

Comment:
please use the `PATest` instead of `E2ETest`

Reply:
@Avilir did you mean PASTest ?

Reply:
yes @ypersky1980 , thanks for correct me